partcad/src/partcad/shape.py

```diff
@@ -133,8 +134,15 @@ def render_svg(
         project=None,
         filepath=None,
     ):
-        _, filepath = self.render_getopts("svg", ".svg", project, filepath)
-        self.render_svg_somewhere(project, filepath)
+        svg_opts, filepath = self.render_getopts("svg", ".svg", project, filepath)
+
+        svg_render_opts = DEFAULT_RENDER_SVG_OPTS.copy()
+
+        svg_render_opts["marginTop"] = svg_opts.get("marginTop", svg_render_opts["marginTop"])
+        svg_render_opts["marginLeft"] = svg_opts.get("marginLeft", svg_render_opts["marginLeft"])
```

Comment:
Too early to apply them here. This way PNG is not influenced. Pass it further down to render_svg_somewhere() as a part of project.ctx

Reply:
I will modify the code that will affect both svg & png. :+1:

Reply:
I will modify the code that will affect both svg & png. :+1: